doc/src/input/pairs.md

```diff
@@ -0,0 +1,166 @@
+# Pair interactions
+
+## Cutoff treatment
+
+When computing the energy and forces for non-bonded pair interactions, Lumol
+uses a cutoff radius $rc$. This means that the force and energy associated with
+any pair at a distance bigger than $rc$ will be zero. We can use two different
+cutoff scheme, presented in the following section.
+
+In the potentials input file, the cutoff should be specified for all the
+`[[pairs]]` sections. It can be specified once for all the pairs in the `global`
+section, and then overridden for specific interactions:
+
+```toml
+# Use a simple cutoff with a radius of 10 A for all the pairs interactions
+[global]
+cutoff = "10 A"
+
+[[pairs]]
+atoms = ["A", "A"]
+lj = {sigma = "3 A", epsilon = "123 kJ/mol"}
+
+[[pairs]]
+atoms = ["B", "B"]
+lj = {sigma = "3 A", epsilon = "123 kJ/mol"}
+
+# Except for this one, use a shifted cutoff with a radius of 8 A
+[[pairs]]
+atoms = ["A", "B"]
+lj = {sigma = "3 A", epsilon = "123 kJ/mol"}
+cutoff = {shifted = "8 A"}
+```
+
+### Simple cutoff
+
+This scheme just set the potential $U(r)$ to zero for any distance bigger than
+$rc$:
+
+$$ V(r) = \begin{cases}
+    U(r) & r <= rc \\\\
+    0 & r > rc
+\end{cases}$$
+
+In the input, this uses a single string containing the cutoff radius as the
+`cutoff` value.
+
+```toml
+[global]
+cutoff = "10 A"
+
+[[pairs]]
+atoms = ["O", "O"]
+lj = {x0 = "3 A", k = "5.9 kJ/mol/A^2"}
+cutoff = "8 A"
+```
+
+### Shifted energy
+
+The potential $U$ can be additionally shifted to make sure it is continuous at
+$r = rc$. This is important for molecular dynamics, where a discontinuity means
+an infinite force in the integration.
+
+$$ V(r) = \begin{cases}
+    U(r) - U(rc) & r <= rc \\\\
+    0 & r > rc
+\end{cases}$$
+
+In the input, this uses a table containing the shifted value, which must be a
+string containing the cutoff radius.
+
+```toml
+[global]
+cutoff = {shifted = "8 A"}
+
+[[pairs]]
+atoms = ["O", "O"]
+lj = {x0 = "3 A", k = "5.9 kJ/mol/A^2"}
+cutoff = {shifted = "10 A"}
+```
+
+### Tail correction
+
+Tail corrections (also called long range corrections) are a way to add back the
+missing energy from using a cutoff distance. If the simulated system is
```

Comment:
Suggestion: ...are a way to account for the error we introduce by cutting off the potential.